client/trino-jdbc/pom.xml

```diff
@@ -286,6 +286,7 @@
                                     <excludes>
                                         <exclude>META-INF/maven/**</exclude>
                                         <exclude>META-INF/services/com.fasterxml.**</exclude>
+                                        <exclude>META-INF/proguard/okhttp3.pro</exclude>
```

Comment:
Was there any conflict here?

Reply:
This is ProGuard config file, presumably useful if you are using ProGuard to minimize OkHttp. We have no need to ship it in the shaded JDBC driver, plus it would not be useful as we repackage OkHttp and thus the config would be wrong anyway. In other words, it's junk we don't need.

Reply:
>  if you are using ProGuard to minimize 

I have mixed feelings about ProGuard, but do you think it makes sense to minimize JDBC driver?

Reply:
the JDBC jar is currently 6.4M. Not a small one, but it used to be much bigger and no-one complained (or at least not very loudly)